ENGDOCS-1733 (#18457)
docker/docs: content/desktop/install/windows-install.md

  - 64-bit processor with [Second Level Address Translation (SLAT)](https://en.wikipedia.org/wiki/Second_Level_Address_Translation)
  - 4GB system RAM

>   - Turn on BIOS-level hardware virtualization support in the
>     BIOS settings. For more information, see

Reviewer: ```suggestion
  - Enable hardware virtualization in BIOS. For more information, see
```
